test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -974,6 +974,8 @@ func doPut(url, content string) (string, error) {
 	return strBody, nil
 }
 
+// ReserveMemory creates a replication controller with pods that each request a
```

Comment:
Just a nit: specified amount of memory ('megabytes' parameter) is total for all replicas, not per pod. Can you please rephrase it? Otherwise LGTM, sorry it takes so long:)

Reply:
Certainly! Great idea, fixing now :)

Reply:
@aleksandra-malinowska fixed :)